src/main/java/net/dv8tion/jda/core/requests/WebSocketClient.java

```diff
@@ -331,6 +333,11 @@ public void close(int code)
         socket.sendClose(code);
     }
 
+    public void close(int code, String reason)
```

Comment:
Does this actually need to be public?

Reply:
It was useful for testing and can be useful for debugging in the future